frontend/public/components/job.tsx

```diff
@@ -138,89 +97,159 @@ const JobTableRow: RowFunction<JobKind> = ({ obj: job, index, key, style }) => {
-            <DetailsItem label="Parallelism" obj={job} path="spec.parallelism" />
-            <DetailsItem
-              label="Active Deadline Seconds"
-              obj={job}
-              path="spec.activeDeadlineSeconds"
-            >
-              {job.spec.activeDeadlineSeconds
-                ? pluralize(job.spec.activeDeadlineSeconds, 'second')
-                : 'Not Configured'}
-            </DetailsItem>
-          </ResourceSummary>
-        </div>
-        <div className="col-md-6">
-          <SectionHeading text="Job Status" />
-          <dl className="co-m-pane__details">
-            <dt>Status</dt>
-            <dd>
-              <Status status={jobStatus(job)} />
-            </dd>
-            <DetailsItem label="Start Time" obj={job} path="status.startTime">
-              <Timestamp timestamp={job.status.startTime} />
-            </DetailsItem>
-            <DetailsItem label="Completion Time" obj={job} path="status.completionTime">
-              <Timestamp timestamp={job.status.completionTime} />
-            </DetailsItem>
-            <DetailsItem
-              label="Succeeded Pods"
-              obj={job}
-              path="status.succeeded"
-              defaultValue="0"
-            />
-            <DetailsItem label="Active Pods" obj={job} path="status.active" defaultValue="0" />
-            <DetailsItem label="Failed Pods" obj={job} path="status.failed" defaultValue="0" />
-          </dl>
+export const JobDetails: React.FC<JobsDetailsProps> = ({ obj: job }) => {
+  const { t } = useTranslation();
+  return (
+    <>
+      <div className="co-m-pane__body">
+        <div className="row">
+          <div className="col-md-6">
+            <SectionHeading text={t('workload~Job details')} />
+            <ResourceSummary resource={job} showPodSelector>
+              <DetailsItem
+                label={t('workload~Desired completions')}
+                obj={job}
+                path="spec.completions"
+              />
+              <DetailsItem label={t('workload~Parallelism')} obj={job} path="spec.parallelism" />
+              <DetailsItem
+                label={t('workload~Active deadline seconds')}
+                obj={job}
+                path="spec.activeDeadlineSeconds"
+              >
+                {job.spec.activeDeadlineSeconds
+                  ? t('workload~second', { count: job.spec.activeDeadlineSeconds })
+                  : t('workload~Not configured')}
+              </DetailsItem>
+            </ResourceSummary>
+          </div>
+          <div className="col-md-6">
+            <SectionHeading text={t('workload~Job status')} />
+            <dl className="co-m-pane__details">
+              <dt>{t('workload~Status')}</dt>
+              <dd>
+                <Status
+                  status={
+                    job?.status
+                      ? job?.status?.conditions?.[0]?.type || t('workload~In progress')
```

Comment:
This will break the icons in other languages since `Status` gets the icon for the text value. I'd leave it as `In progress` without translating for now since other statuses will not be translated. (We'll need to update the `Status` component to handle the new case.)